todo_app/data/todo.py

```diff
@@ -0,0 +1,106 @@
+from flask import session
+
+from todo_app.data.Item import Item
+import requests, os
+
+TRELLO_API_KEY = os.environ.get('TRELLO_API_KEY')
+TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
+TRELLO_BOARD_ID = "608c0972ee945e2cd7fb1211"
+TRELLO_BOARD_TODO_LIST_ID = "608c0972ee945e2cd7fb1212"
+TRELLO_BOARD_DONE_LIST_ID = "608c0972ee945e2cd7fb1214"
```

Comment:
It's nice to store these in .env as well, with instructions on how to set them up in README, so that a new user only has to change .env and not the python code to get set up,